docs/development.md

```diff
@@ -70,7 +70,22 @@ go run . serve
 
 ## Windows (ARM)
 
-Windows ARM does not support additional acceleration libraries at this time.
+To build Ollama on Windows ARM you will need a `gcc` compatible compiler in your PATH.  An official GCC release is currently not available on Windows ARM, but a `gcc` compatibility CLI is available for clang.
```

Comment:
We mention this above and link to `llvm-mingw` - would we still need this section?